index.js

```diff
@@ -1,5 +1,6 @@
 const { App, WorkflowStep } = require('@slack/bolt');
-const blocks = require('./block.js');
+const blocks = require('./blocks/inputClub.js');
+require('dotenv').config();
```

Comment:
```suggestion
const dotenv = require('dotenv').config();

Object.keys(dotenv).forEach(key => {
  process.env[key] = dotenv[key];
});
```

こうだったかな？すると nodemon などの dev サーバー立ててる時に env 内の内容を変えても内容を更新できます